### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'modifier')

### ISSUE THREAD

**Issue by TheOceanBreeze** [tailwindlabs/tailwindcss-intellisense#641]
What version of Tailwind CSS are you using?
3.2.0

What build tool (or framework if it abstracts the build tool) are you using?
VSCodium 1.72.2 with Tailwind CSS IntelliSense v0.8.7 (I'm also using Hugo and PostCSS)

What version of Node.js are you using?
Node v18.9.0 (NPM v8.19.1)

What browser are you using?
Firefox

What operating system are you using?
Linux

Reproduction URL
N/A

Describe your issue
When trying to use the IntelliSense plugin in VSCodium, the plugin crashes with a TypeError.

```
Found Tailwind CSS config file: /home/go-rs/ghq/github.com/69-Studios/69-Learn/tailwind.config.js
Loaded postcss v8.4.18: /home/go-rs/ghq/github.com/69-Studios/69-Learn/node_modules/postcss
Loaded tailwindcss v3.2.0: /home/go-rs/ghq/github.com/69-Studios/69-Learn/node_modules/tailwindcss
TypeError: Cannot read properties of undefined (reading 'modifier')
    at api.addVariant (/home/go-rs/ghq/github.com/69-Studios/69-Learn/node_modules/tailwindcss/lib/lib/setupContextUtils.js:578:40)
    at /home/go-rs/ghq/github.com/69-Studios/69-Learn/node_modules/tailwindcss/lib/lib/setupContextUtils.js:538:38
    at /home/go-rs/.vscode-oss/extensions/bradlc.vscode-tailwindcss-0.8.7-universal/dist/tailwindServer.js:695:4431
    at Array.forEach (<anonymous>)
    at Jwe (/home/go-rs/.vscode-oss/extensions/bradlc.vscode-tailwindcss-0.8.7-universal/dist/tailwindServer.js:695:3916)
    at /home/go-rs/.vscode-oss/extensions/bradlc.vscode-tailwindcss-0.8.7-universal/dist/tailwindServer.js:695:286
    at Generator.next (<anonymous>)
    at i (/home/go-rs/.vscode-oss/extensions/bradlc.vscode-tailwindcss-0.8.7-universal/dist/tailwindServer.js:1:1059)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
```

This is my tailwind.config.js (However it appears that the bug happens on a clean slate version of tailwind.config.js):

```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ['./content/**/*.{html,js}',
  './themes/69lt/layouts/**/*.{html,js}'
],
  theme: {
    extend: {},
  },
  plugins: [],
}
```

**Comment by bradlc** [comment]
Hey @TheOceanBreeze, this issue is resolved since v0.9.0 of the extension, so if you upgrade you should be good to go 👍

**Comment by abhijeet-619** [comment]
Thanks,, just update VS code and resolved this issue.
